pre-commit-hooks.Rproj

```diff
@@ -15,3 +15,4 @@ LaTeX: pdfLaTeX
 BuildType: Package
 PackageUseDevtools: Yes
 PackageInstallArgs: --no-multiarch --with-keep.source
+PackageRoxygenize: rd,collate,namespace
```

Comment:
What does this again?

Reply:
Got it generated after ticking "Generate documentation with Roxygen" in Project Options in RStudio.

RStudio release "Double Marigold" (e4a1c219, 2020-04-01)
R 3.6.3
devtools 2.3.0
roxygen2 7.1.0

Reply:
Otherwise it I had an issue with passing `@param` with `@inheritParams` in a single function - `use_precommit`.

> checking for code/documentation mismatches ... WARNING
  Codoc mismatches from documentation object 'use_precommit':
  use_precommit
    Code: function(config_source = getOption("precommit.config_source"),
                   force = FALSE, open = rstudioapi::isAvailable(),
                   install_hooks = TRUE, root = here::here())
    Docs: function(config_source = getOption("precommit.config_source"),
                   force = FALSE, open = rstudioapi::isAvailable(), root
                   = here::here())
    Argument names in code not in docs:
      install_hooks
    Mismatches in argument names:
      Position: 4 Code: install_hooks Docs: root

Reply:
Ok, fine.